src/windows/video_widget.py

```diff
@@ -509,15 +509,23 @@ def paintEvent(self, event, *args):
     def centeredViewport(self, width, height):
         """ Calculate size of viewport to maintain aspect ratio """
 
-        window_size = QSizeF(width, height)
-        window_rect = QRectF(QPointF(0, 0), window_size)
+        # Calculate padding
```

Comment:
Not sure why these changes are showing different than develop (since I already merged them)

Reply:
:thinking: